tests/php/modules/shortcodes/test-class.loom.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
Since your provider is declared in `modules/shortcodes/others.php`, I think you could move that test to `tests/php/modules/shortcodes/test-class.others.php` with the rest of the "other" tests.

Reply:
✅